docs/Migrating to Newer SDL Versions/Migrating SDL Core 6.1 to 7.0/index.md

```diff
@@ -0,0 +1,276 @@
+        <description>
+            Image to be displayed for the corresponding alert. See Image. 
+            If omitted, no (or the default if applicable) icon should be displayed.
+        </description>
+    </param>
+    <param name="duration" type="Integer" mandatory="false" minvalue="3000" maxvalue="10000">
+        <description>Timeout in milliseconds.</description>
+    </param>
+    <param name="softButtons" type="Common.SoftButton" mandatory="false" minsize="0" maxsize="2" array="true">
+        <description>App defined SoftButtons</description>
+    </param>
+    <param name="alertType" type="Common.AlertType" mandatory="true">
+        <description>Defines if only UI or BOTH portions of the Alert request are being sent to HMI Side</description>
+    </param>
+    <param name="appID" type="Integer" mandatory="true">
+        <description>ID of application requested this RPC.</description>
+    </param>
+    <param name="cancelID" type="Integer" mandatory="false">
+        <description>
+            An ID for this specific alert to allow cancellation through the `CancelInteraction` RPC.
+        </description>
+    </param>
+</function>
+
+<function name="SubtleAlert" messagetype="response">
+    <param name="tryAgainTime" type="Integer" mandatory="false" minvalue="0" maxvalue="2000000000">
+        <description>Amount of time (in milliseconds) that SDL must wait before resending an alert. Must be provided if another system event or overlay currently has a higher priority than this alert.</description>
+    </param>
+</function>
+
+<function name="OnSubtleAlertPressed" messagetype="notification">
+    <description>
+    Sent when the alert itself is touched (outside of a soft button). Touching (or otherwise selecting) the alert should open the app before sending this notification.
+    </description>
+    <param name="appID" type="Integer" mandatory="true">
+        <description>ID of application that is related to this RPC.</description>
+    </param>
+</function>
+```
+
+### Webengine Projection Support
+
+A new `WEB_VIEW` appHMIType and template layout was added which will allow apps to render a template independent view in a browser environment with javascript and HTML.
+
+```
+<enum name="AppHMIType">
+  <description>Enumeration listing possible app types.</description>
+...
+   <element name="WEB_VIEW" />
+</enum>
+```
+
+This appHMIType must be explicitly specified in an app's policy table entry for SDL Core to allow it to be used.
+
+Policy Table Entry:
+
+```
+...
+"app_policies": {
+    "webengine_appID": {
++       "appHMIType: ["WEB_VIEW"],
+        "keep_context": false,
+        "steal_focus": false,
+        "priority": "NONE",
+        "default_hmi": "NONE",
+        "groups": [
+            "Base-4"
+        ],
+        "RequestType": [],
+        "RequestSubType": []
+    },
+...
+```
+
+
+
+### New UI.GetCapabilities parameter: `pcmStreamCapabilities`
```

Comment:
In previous headers (ie: Deprecated HMI RPC: OnFindApplications) , you didn't include code blocks. For consistency, I think you should either add code blocks to params in other headers, or remove them from this one.

Reply:
fixed